basilisk/tests/test_prompt_attachments_panel.py

```
@@ -0,0 +1,669 @@
+import pytest
```

Ruff (0.11.9)

Missing docstring in public module

(D100)